CyrillicFilters/RussianFilter/sections/specific.txt

```diff
@@ -9994,6 +9994,9 @@ gdetraffic.com,playua.net##.bg-link
 newsyou.info##div[id^="adpartner-"]
 bragazeta.ru,go64.ru,news29.ru,fc-amkar.org,sxem.org##div[id^="banner"]
 afftimes.com,pravda.ru##div[class^="branding_"]
+senkuro.com##.main-section > div.empty:not(#style_important)
+senkuro.com##.main-section > a[target="_blank"]:not(#style_important)
+||https://mirai.senkuro.net/system/ads/
```

Comment:
Rules work fine without `:not(#style_important)` so I think that it can be removed.
The last rule doesn't work with `https://`, it should be - `||mirai.senkuro.net/system/ads/`
```suggestion
senkuro.com##.main-section > div.empty
senkuro.com##.main-section > a[target="_blank"]
||mirai.senkuro.net/system/ads/
```